archive/cpio_old_le.ksy

```diff
@@ -0,0 +1,64 @@
+meta:
+  id: cpio_old_le
+  title: cpio archive, old binary variant, little-endian
+  file-extension: cpio
+  license: CC0-1.0
+  endian: le
+  encoding: ASCII

```

Comment:
Yeah, that's probably a good idea. I mean, we should have multiple ways to display a byte array to begin with. The obvious ones are:

* `"foo\0bar\n\xC7"` - C-style, ASCII shown directly, everything else is C-like encoded
* `"foo\x00bar\x0A\xC7"` - C-style, ASCII shown directly, everything else is hex-encoded
* `66 6F 6F 00 62 61 72 0A C7` - hex dump
* `[102, 111, 111, 0, 98, 97, 114, 10, 199]` - array of decimal integers (probably least useful, but still might prove be of some use in uncommon cases)

Bonus points in C-like strings would have color highlighting for special characters ;)

And, yeah, probably we'll need a way to:

1. Specify "default" visualization in the .ksy
2. Experiment with switching representations on the fly using some sort of menu / buttons for one particular field